sdk/keyvault/tests.yml

```diff
@@ -16,7 +16,6 @@ stages:
         ServiceDirectory: keyvault
         BuildTargetingString: ${{ service }}
         JobName: ${{ replace(service, '-', '_') }}
-        TestTimeoutInMinutes: 240
```

Comment:
You sure you don't need this? You probably copied from .NET, in which case we need it with over 20,000 tests.

Reply:
There are occasionally China cloud platforms that'll run for 1.5-2 hours, but the default timeout of 120 minutes should be sufficient aside from outliers 🙂